larq/math.py

```diff
@@ -0,0 +1,8 @@
+"""Math operations that specific to extremely quantized networks."""
+
+import tensorflow as tf
+
+
+def sign(x):
+    """A sign function that will never be zero"""
```

Comment:
I think we should add this to our public documentation, since it can be quite useful.
For this it would be great to add `Arguments` and `Returns` to the docsting.

Also add it here:
https://github.com/plumerai/larq/blob/f713c6ba741dc1c0267a3c0c4a0cc5c964ff4dda/apidocs.yml#L13-L14
and here:
https://github.com/plumerai/larq/blob/f713c6ba741dc1c0267a3c0c4a0cc5c964ff4dda/mkdocs.yml#L12-L19

For it to show up in the build docs.